spec/traits.dd

```diff
@@ -199,6 +200,31 @@ $(H2 $(GNAME isFuture))
     marked with the `@future` keyword, otherwise `false`. Currently, only
     functions and variable declarations have support for the `@future` keyword.)
 
+$(H2 $(GNAME isDisabled))
+
+    $(P Takes one argument and returns `true` if it's a function declaration
+    marked `@disable`.)
+
+---
+struct Foo
+{
+    @disable void foo();
+    void bar(){}
+}
+
+static assert(__traits(isDisabled, Foo.foo));
+static assert(!__traits(isDisabled, Foo.bar));
+---
+
+    $(P For any other declaration and even if it can be marked `@disable`,
+    `false` is returned because the annotation is a no-operation.)
```

Comment:
because the annotation is a no-operation
->
because the annotation has no effect